lib/razor/cli/navigate.rb

```diff
@@ -5,14 +5,39 @@
 
 module Razor::CLI
   class Navigate
+    PRIORITY_RAZOR_API = "https://localhost:8151/api"
```

Comment:
I would rename these to
```
PE_RAZOR_API
FOSS_RAZOR_API
```

(and maybe a comment clarifying what those variables are for). Bit weird to have DEFAULT_RAZOR_API when it might not be the default (for pe installations), and priority doesn't make it clear that the URL is for PE razor.

Reply:
The attempt here is to keep references to PE out of the codebase. From purely a functionality perspective, the only difference between the two URLs is that one takes priority over the other one, which is why I named one "Priority".

Reply:
But the whole reason we have the priority endpoint is b/c that's the PE one, so why would it not be a good thing to include that info. here?

Reply:
From an architectural perspective, the reason PE shouldn't be in this codebase is because the product stands alone, and PE simply adds functionality to it. We want to avoid coupling from FOSS in the direction of PE.

Reply:
I included a bit of detail re: PE in the commit message, which is the reason why we are making the change. The 'why' behind adding the Priority functionality is that we want to check one HTTPS/TLS URL before we check another HTTP/non-TLS URL.

Reply:
What about naming them to
```
HTTPS_RAZOR_API
HTTP_RAZOR_API
```

That would make it clearer and also enforce the de-coupling of PE.

Priority and default are still kind of weird, b/c you can have the priority be the default, which would mean that the default razor api is not the default, if that makes sense.

Reply:
ooh I like that. because then it'll also be consistent with razor-server.

Reply:
Choosing `RAZOR_HTTPS_API` and `RAZOR_HTTP_API` for consistency with environment variables used in PE.